### ISSUE TITLE
请教代码问题

### ISSUE THREAD

**Issue by maizi0059** [liuyuemaicha/Adversarial-Learning-for-Neural-Dialogue-Generation-in-Tensorflow#23]
line 214：
# 4.Update G on (X, ^Y ) using reward r

gan_adjusted_loss, gen_step_loss, _ =gen_model.step(sess, encoder, decoder, weights, bucket_id, forward_only=False, reward=reward, up_reward=True, debug=True)
gen_loss += gen_step_loss / gen_config.steps_per_checkpoint

这里的encoder, decoder, weights应该指的是(X,Y)，而不是(X, ^Y )，这部分是不是应该修改为(X, ^Y )的输入？我想请问下，这部分是我理解错了么？
